src/app/mesh/qgsmeshlayerproperties.h

```diff
@@ -90,6 +90,8 @@ class APP_EXPORT QgsMeshLayerProperties : public QgsOptionsDialogBase, private U
      * was loaded but dialog is canceled */
     QgsMapLayerStyle mOldStyle;
 
+    bool mIsMapSettingsTemporal = false;
```

Comment:
this is assigned in constructor of QgsMeshLayerProperties but I do not see where else it is used or changed afterwards? maybe we do not need the member variable at all ?

Reply:
here : https://github.com/qgis/QGIS/blob/3fc1cb126d2767021a47309718ab91180e437464/src/app/mesh/qgsmeshlayerproperties.cpp#L223

and here : 

https://github.com/qgis/QGIS/blob/3fc1cb126d2767021a47309718ab91180e437464/src/app/mesh/qgsmeshlayerproperties.cpp#L474

Reply:
this is where is it used, but where it is changed except contructor?

Reply:
nowhere. I don't see the problem...